ui/src/components/SampleGrid/utils.js

```diff
@@ -0,0 +1,9 @@
+import { useSelector } from 'react-redux';
+
+export function useComponent(name) {
+  return useSelector((state) =>
```

Comment:
Copied from the samples control util class. this can be useful for other component related to sample List.